src/Bio/VCF/Parser/Parser.hs

```diff
@@ -1,22 +1,58 @@
 {-# LANGUAGE OverloadedStrings #-}
+
 module Bio.VCF.Parser.Parser where
 
 import Data.Attoparsec.ByteString
-import Data.Attoparsec.ByteString.Char8
+import qualified Data.Attoparsec.ByteString.Char8 as AC8
```

Comment:
We want performance, right? Attoparsec is very fast but has terrible error messages.